src/main/java/ui/MenuControl.java

```diff
@@ -182,6 +208,9 @@ private void onBoardDelete(String boardName) {
 
         if (response.isPresent() && response.get().getButtonData() == ButtonData.OK_DONE) {
             prefs.removeBoard(boardName);
+            if (prefs.getLastOpenBoard().equals(boardName)) {
+                prefs.setLastOpenBoard("");
```

Comment:
This is where that `clear` method would be useful.

We also have to handle the possibility of the last-opened board not being present.